CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.1.0]
+### Changed
+- Confirm a transaction conditionally ([#301](https://github.com/MetaMask/smart-transactions-controller/pull/301))
+- meta: Drop nodejs v16 support ([#277](https://github.com/MetaMask/smart-transactions-controller/pull/277))
```

Comment:
Dropping Node 16 support already happened in 7.0.0 — this was just an update to the README and a dev dependency — so we don't need to include it.

``` suggestion
```

Reply:
It was automatically added in the changelog, but sure, if it was added in the previous release, we can remove it.

Reply:
That's expected; the action that creates the release PR doesn't know what to add and what not to add, so it adds everything by default. This gives us a set of PRs to review and then we can opt to take things out if need be.